modules/040-control-plane-manager/monitoring/prometheus-rules/control-plane-manager.yaml

```diff
@@ -47,6 +47,23 @@
       description: |-
         Current kubernetes version "{{ $labels.k8s_version }}" is deprecated, and its support will be removed within 6 months
 
-        Please migrate to the next kubernetes version (at least 1.22)
-        
+        Please migrate to the next kubernetes version (at least 1.23)
+
+        Check how to update the Kubernetes version in the cluster here - https://deckhouse.io/documentation/deckhouse-faq.html#how-do-i-upgrade-the-kubernetes-version-in-a-cluster
+
+  - alert: D8KubernetesVersionEndOfLife
+    for: 10m
+    expr: max by (k8s_version) (d8_kubernetes_version{k8s_version="1.21"}) == 1
+    labels:
+      severity_level: "4"
+      tier: cluster
+    annotations:
+      plk_protocol_version: "1"
+      plk_markup_format: "markdown"
+      summary: Kubernetes version "{{ $labels.k8s_version }}" has reached End Of Life.
```

Comment:
Also, `plk_protocol_version`, `plk_markup_format` and `summary` fields are missed in the first alert for some reason.